crowbartools/Firebot
# #2187 V5 pronoun variable

handles #2187

### Description of the Change
Creates a new replace variable that can pull pronoun information for a user from the pronoun site. It also allows the user to put in a fallback, as well as define which pronoun to use from a set like "They/Them".

$pronouns[$username, 0, they/them] = Returns "she/her" with fallback of "they/them"
$pronouns[$username, 1, they] = Returns "she" with fallback of "they"
$pronouns[$username, 2, them] = Returns "her" with fallback of "them"

### Applicable Issues
This won't cover every single possible combination of pronouns people might need of course. For example "his" is not something on the pronoun site, so people will need to get creative to cover some edge cases.


### Testing
- Create a command in firebot using the new variable.
- Run command in chat.
- Try putting gibberish in for the username to make sure fallbacks are used.
